Statistics.cs

```diff
@@ -64,29 +69,87 @@ public void CompareCitiesPopulation()
 
         }
 
-        public void ShowCityOnMap()
+        /// <summary>
+        /// Show a location on the google maps site
+        /// </summary>
+        /// <param name="lat">Lattitude of the location</param>
+        /// <param name="lng">Longitude of the location</param>
+        public void ShowCityOnMap(double lat, double lng)
         {
-
+            string url = $"https://www.google.com/maps/@{lat},{lng},15z";
+            try
+            {
+                System.Diagnostics.Process.Start("explorer.exe", url);
```

Comment:
this didn't work at all for me. I did some googling though and this line seems to correctly do what we want
`Process.Start(new ProcessStartInfo("cmd", $"/c start {url}") { CreateNoWindow = true });`

Reply:
If we want to get super into it we can do the full check for OS too. It talks about it at https://brockallen.com/2016/09/24/process-start-for-urls-on-net-core/

Reply:
The Process.Start command is a absolute mess. Originally I was going to just use chrome.exe or just pass a standard url but that didnt work on my computer at all. This is the only one I managed to find that worked in the slightest but is dependent on having microsoft edge installed. I will look into implementing the one you showed later tonight.